### ISSUE TITLE
bug: can't login after creating user in self hosted

### ISSUE THREAD

**Issue by Akanshu9990** [uxbox/uxbox#3944]
### Steps To Reproduce

1. install docker in ubuntu server
2. download docker-compose file
3. run the containers
4. create a user
5. login with user details

### Expected behavior

user must be logged in

### Actual behavior

wrong email or password

**Comment by christian-list** [comment]
I have the same issue, when I try to execute penp. I did following steps:

1. Copied docker-compose.yaml in local directory
2. In line 96 and 140 I added _disable-secure-session-cookies_ to the PENPOT FLAGS (later I tried only to add this to the backend -> line 140)
3. Executed _docker compose -p penpot -f docker-compose.yaml up -d_
4. With the docker ui I opend a shell into the penpot-backend container
5. Executed following command: ./manage.py create-profile -n **** -e ***@***.** -p **** (used the same password in the web version)
6. Got response: Created: ***@***.** / SomeHash
7. Called localhost:9001 and Try to login
8. Got response 400 with the response body ["^ ","~:type","~:validation","~:code","~:wrong-credentials"]

Additional info:
Tested in Chrome, Firefox and Safari
Tested with Docker Desktop 4.25 and 4.26.1

Environment:
Mac M1 Pro 32GB
Sonoma 14.1.1

**Comment by aw042** [comment]
I have the same issue. I was able to configure it so that demo users were enabled, but when I try to create a demo user it says something went wrong (backend error). Not really sure what to do, I tried to configure the postgres database but I might have done something wrong. It would be great if there was a video that walked through the whole self hosting process.

**Comment by kristapsx** [comment]
So what I figured out, that to bypass"Email or password is incorrect", need select forgot password and recover it. All mails are sent to mailcatcher, so opening http://localhost:1080/ will access message and after entering new credentials, problem will be temporary solved. But login works only on local host, or will get - "Something wrong has happened.". For now I am trying to mess with configuration, but without success. Seems like in lines starting from about 60 can fix that issues, simply setting up https, but it wont take any effect so far I tried. So lets see, now trying to set up proxy, for local environment little overkill, but maybe that can fix things.

Got it working. So, no need for proxy or https setups, but if penpot already is running on docker, then need remove all containers, then change in docker-compose.yaml PENPOT_PUBLIC_URI to http://0.0.0.0:9001 and then again recreate docker setup. And PENPOT_FLAGS in both places need to be appended with disable-secure-session-cookies

**Comment by chendelin1982** [comment]
yes, **disable-secure-session-cookie** if very import if not use https

**Comment by yamila-moreno** [comment]
Hi folks!

On a general note, we recommend checking the [official documentation](https://help.penpot.app/technical-guide/configuration/) and understanding the configuration flags. We can help if something is not clear.

Now, regarding some issues were metioned in this thread:
- demo users: [this specific documentation](https://help.penpot.app/technical-guide/configuration/#demo-environment) should be helpful to understand how to configure them
- regarding http/https. The recommended way to use Penpot is with HTTPs. Nonetheless, [this specific documentation](https://help.penpot.app/technical-guide/configuration/#penpot-uri) should cover the needs to configure it knowingly
- emails: depending on your goals with the self-hosted installation, you may want to use the console, or the mail-catcher or a proper [smtp service](https://help.penpot.app/technical-guide/configuration/#email-configuration).
- if you want to create a user using the `manage.py` script, ensure you also have the `disable-email-confirmation` flag.

As there are different issues covered, I'm closing this thread. Feel free to open new ones if you find other issues.

Cheers!

pd. and ensure you're using the latest Penpot version
